src/frontend/shared-components/src/modules/termsOfUseTestData.js

```diff
@@ -0,0 +1,308 @@
+import React from "react";
+
+const content = (
+  <>
+    <p>
+      In these Terms of Use, &quot;you&quot; or &quot;your&quot; includes the
+      individual using or accessing the Electronic Criminal Record Check Portal
+      (the &quot;Site&quot;) on their own behalf to submit an online request for
+      a criminal record check.
+    </p>
+    <p>
+      These Terms of Use are an agreement between you and Her Majesty the Queen
+      in Right of the Province of British Columbia, represented by the Minister
```

Comment:
Should we use real data for this test or just fake the Terms ?

Reply:
either one should be fine, this is not the official terms of use anyways - the real text will come in actual app not in shared-component - this is just for the storybook for other people to see what it looks like to consume it